src/screens/account.tsx

```diff
@@ -245,6 +245,9 @@ export const AccountPage = () => {
 	if (isHotkey) {
 		return <Navigate to={`/hotkey/${address}`} replace />;
 	}
+	if (isColdkey) {
+		return <Navigate to={`/coldkey/${address}`} replace />;
+	}
```

Comment:
I don't think this is right.
Users should be able to check the account balance and other necessary information.

Reply:
Removed